python/lsst/cp/pipe/ptc/astierCovPtcFit.py

```diff
@@ -35,32 +35,32 @@ def makeCovArray(inputTuple, maxRangeFromTuple=8):
 
     Parameters
     ----------
-    inputTuple: `numpy.ndarray`
+    inputTuple : `numpy.ndarray`
         Structured array with rows with at least
         (mu, afwVar, cov, var, i, j, npix), where:
 
         mu : 0.5*(m1 + m2), where:
             mu1: mean value of flat1
             mu2: mean value of flat2
-        afwVar: variance of difference flat, calculated with afw
-        cov: covariance value at lag(i, j)
-        var: variance(covariance value at lag(0, 0))
-        i: lag dimension
-        j: lag dimension
-        npix: number of pixels used for covariance calculation.
-
-    maxRangeFromTuple: `int`
+        afwVar : variance of difference flat, calculated with afw
+        cov : covariance value at lag(i, j)
+        var : variance(covariance value at lag(0, 0))
+        i : lag dimension
+        j : lag dimension
+        npix : number of pixels used for covariance calculation.
```

Comment:
I don't think these will format in a way numpydoc can parse (or at all, given that there's no list formatting). Accident of search-and-replace?

Reply:
Updated to be standard.